modules/account-lib/src/coin/cspr/transaction.ts

```diff
@@ -26,7 +27,11 @@ export class Transaction extends BaseTransaction {
     if (!keys.prv) {
       throw new SigningError('Missing private key');
     }
-    if (this._deploy.approvals.some(ap => !ap.signer.startsWith(SECP256K1_PREFIX))) {
+    if (
+      this._deploy.approvals.some(
+        ap => !ap.signer.startsWith(SECP256K1_PREFIX || !isValidPublicKey(ap.signer.slice(2))),
+      )
+    ) {
```

Comment:
> The logic looks good.
> Since [STLX-1715](https://bitgoinc.atlassian.net/browse/STLX-1715) is part of this change, and [STLX-1662](https://bitgoinc.atlassian.net/browse/STLX-1662) has been resolved, can we include the change to get the owners from the deploy in this PR?

Hi @CapnMigraine .

We've analyzed the required changes for the refactor after updating to 1.0.19 and we think it's safer to do the refactor later so we can test this story and close it before the end of this sprint. Otherwise we'll not be able to do it.

@rliberal-simtlix , can you please add the new Jira Ticket for the refactor to this PR? 
Thanks!

Reply:
> Are the closing parens in this check correct?
> Looks like there should be one closing after `SECP256K1_PREFIX`
> 
> Also, style-nit:

Regarding this change, I can do this change. However, ESLint is recommending me to format the code this way.